### PR TITLE
[SYCL][ESIMD] Fix return value for test

### PR DESCRIPTION
**Pull request by sarnex**
Right now it's returning 1 so the test is failing.

### CODE REVIEW

**Comment by sarnex** [comment]
@intel/llvm-reviewers-runtime Hey all, can we merge this ASAP? It's trivial and the test is currently failing in post-commit. Thanks.

**Review by againull — APPROVED** [review]

File: sycl/test-e2e/InvokeSimd/Spec/simd_mask_merge.cpp [hunk 1]
```diff
@@ -100,5 +100,5 @@ int main() {
   }
 
   std::cout << (err_cnt > 0 ? "FAILED\n" : "Passed\n");
-  return err_cnt == 0;
+  return err_cnt != 0;
```
Comment (aelovikov-intel) [inline sycl/test-e2e/InvokeSimd/Spec/simd_mask_merge.cpp:103]: I think this would be better:
```suggestion
  assert(err_cnt == 0);
  return 0;
```

**Comment by aelovikov-intel** [comment]
How did it pass pre-commit? Is the configuration specific to post-commit?

**Comment by sarnex** [comment]
> How did it pass pre-commit? Is the configuration specific to post-commit?

it didn't even run on any hardware, i think there's a problem with the ci. I'll ping the devlops people in the PR

**Review by v-klochkov — APPROVED** [review]

File: sycl/test-e2e/InvokeSimd/Spec/simd_mask_merge.cpp (hunk 1 above)
Comment (sarnex) [reply sycl/test-e2e/InvokeSimd/Spec/simd_mask_merge.cpp]: done in latest commit, thx

**Comment by sarnex** [comment]
thx for fast turnaround

File: sycl/test-e2e/InvokeSimd/Spec/simd_mask_merge.cpp (hunk 1 above)
Comment (v-klochkov) [reply sycl/test-e2e/InvokeSimd/Spec/simd_mask_merge.cpp]: I actually think the original version was better. 
If somehow (by additional options maybe) the assert are turned off, then the test would return pass.

Comment (sarnex) [reply sycl/test-e2e/InvokeSimd/Spec/simd_mask_merge.cpp]: i can change it back in another pr if @aelovikov-intel is okay with it

**Comment by aelovikov-intel** [comment]
We have so many tests that rely on the assert (https://github.com/search?q=repo%3Aintel%2Fllvm+assert+path%3Asycl%2Ftest-e2e&type=code) that I think this argument is moot.

File: sycl/test-e2e/InvokeSimd/Spec/simd_mask_merge.cpp (hunk 1 above)
Comment (aelovikov-intel) [reply sycl/test-e2e/InvokeSimd/Spec/simd_mask_merge.cpp]: We have so many tests that rely on the assert (https://github.com/search?q=repo%3Aintel%2Fllvm+assert+path%3Asycl%2Ftest-e2e&type=code) that I think this argument is moot.

**Pull request merged**